src/Experimental/Form.fs

```diff
@@ -39,12 +39,12 @@ type Form<'a> = Form of FormProp<'a> list * ServerSideValidation<'a> list
 
 let formatDec (d : Decimal) = d.ToString(Globalization.CultureInfo.InvariantCulture)
 
-#if NETSTANDARD1_5
+#if NETSTANDARD2_0
 open System.Reflection
 #endif
 
 let (|Optional|_|) (typ : Type) =
-  #if NETSTANDARD1_5
+  #if NETSTANDARD2_0
```

Comment:
#else part of this #if can be compiled under netstandard 2.0, so this code could be removed